lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -68,8 +101,190 @@
+      background-position: 0 0;
+    }
+  }
+}
+
+// editing
+.annotator-editor {
+
+  .annotator-controls {
+    @include text-align(left);
+    @include clearfix();
+    background: $notes-annotator-background-med !important; //matches annotator JS editing bubble triangle color
+    font-family: $f-sans-serif !important;
+    padding: 8px;
+    border: none !important;
+    border-radius: 0 !important;
+
+    // actions
+    .annotator-save, .annotator-cancel {
+      @extend %notes-reset-background;
+      font-family: $f-sans-serif !important;
+      font-size: 14px !important;
+      padding: ($baseline/4) ($baseline/2) !important;
+      border: none;
+      box-shadow: none;
+      text-shadow: none !important;
+
+      // removing vendor icons
+      &:after {
+        display: none !important;
+      }
+    }
+
+    .annotator-save {
```

Comment:
NOTE: fudging the left alignment and visual ordering of AnnotatorJS's editing actions :/